### PR TITLE
Day twelve

### PR DESCRIPTION
**Pull request by Talaspaev**

### CODE REVIEW

File: src/Components/Dialogs/dialogsContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import { addMessageActionCreator, updateMessageActionCreator } from '../../redux/redux-state';
```
Comment (Dmitry-White) [inline src/Components/Dialogs/dialogsContainer.jsx:2]: There's an empty line missing here.

File: src/Components/Users/usersContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+import { followUserActionCreator, unfollowUserActionCreator, setUserActionCreator } from '../../redux/redux-state';
```
Comment (Dmitry-White) [inline src/Components/Users/usersContainer.jsx:2]: There's an empty line missing here.

File: src/index.js [hunk 3]
```diff
@@ -1,20 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
```
Comment (Dmitry-White) [inline src/index.js:3]: There's a redundant empty line.

File: src/redux/redux-state.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { createStore, combineReducers } from 'redux';
+
+import profileReducer from './profileReducer';
```
Comment (Dmitry-White) [inline src/redux/redux-state.js:3]: There import groups order is incorrect here.

File: src/Components/Dialogs/dialogsContainer.jsx (hunk 1 above)
Comment (Talaspaev) [reply src/Components/Dialogs/dialogsContainer.jsx:3]: i've fixed missing lines

File: src/Components/Users/usersContainer.jsx (hunk 2 above)
Comment (Talaspaev) [reply src/Components/Users/usersContainer.jsx:3]: same above

File: src/index.js (hunk 3 above)
Comment (Talaspaev) [reply src/index.js]: same above

File: src/redux/redux-state.js (hunk 4 above)
Comment (Talaspaev) [reply src/redux/redux-state.js:6]: same above

**Pull request merged**